addons/hr/models/hr_employee_base.py

```diff
@@ -59,6 +59,7 @@ class HrEmployeeBase(models.AbstractModel):
         ('presence_undetermined', 'Undetermined')], compute='_compute_presence_icon')
     show_hr_icon_display = fields.Boolean(compute='_compute_presence_icon')
     newly_hired = fields.Boolean('Newly Hired', compute='_compute_newly_hired', search='_search_newly_hired')
+    employee_properties = fields.Properties('Properties', definition='department_id.employee_properties_definition', copy=True)
```

Comment:
This will expose the properties to the ``hr.employee.public`` model, which is exposed to all internal employees (not only HR officer).

It seems to be an undesired side effect.

Reply:
Seems not anymore. Things go fast.